packages/plugin-fastlane/src/index.ts

```diff
@@ -47,6 +108,29 @@ eval_gemfile(plugins_path) if File.exist?(plugins_path)
 `
       );
     }
+
+    if (!config.android.signing) {
+      throw Error(
+        "[CodePluginFastlane]: missing android signing configuration"
+      );
+    }
+
+    await fs.copyFile(
+      pathk.config.resolve(config.android.signing.storeFile),
+      pathk.project.resolve("android", "app", "release.keystore")
+    );
+
+    await fsk.update(
+      pathk.project.resolve("android", "app", "build.gradle"),
+      /(signingConfigs[.\s\S]+?release \{)[.\s\S]+?\}/m,
+      `$1
+            storeFile file('release.keystore')
+            storePassword System.getenv("STORE_PASSWORD")
+            keyAlias '${config.android.signing.keyAlias}'
+            keyPassword System.getenv("KEY_PASSWORD")
+        }
```

Comment:
@crherman7 I am ending up with an invalid app/build.gradle in example app project after init, adding the closing bracket here causes a duplicate one cause there currently is an[ if statement](https://github.com/brandingbrand/flagship/blob/main/packages/core/assets/template/android/app/build.gradle#L120) inside release signingConfig and we remove the if block but the closing bracket from release still exists

Reply:
good catch updated